console/accounts_profile.go

```diff
@@ -0,0 +1,65 @@
+/*
```

Comment:
Please rename the command as account_profile since that is the structure behind.